layouts/partials/episode-module.html

```diff
@@ -1,116 +1,90 @@
 {{/* Set some variables based on if its a episode/bonus/trailer */}}
 {{/* This will need updating if we go above 9 seasons or episodes */}}
-{{ if in .Params.seasonepisode "B" }}
+{{ if in .context.Params.seasonepisode "B" }}
     {{/* Bonus episode */}}
-    {{ $.Scratch.Set "episodeNumber" (printf "Bonus Episode") }}
-    {{ $.Scratch.Set "imageFolder" (printf "S0%sB0%s" (string .Params.season) (strings.TrimPrefix "B" .Params.seasonepisode) ) }}
-{{ else if eq (string .Params.seasonepisode) (string "0") }}
+    {{ .context.Scratch.Set "episodeNumber" (printf "Bonus") }}
+    {{ .context.Scratch.Set "imageFolder" (printf "S0%sB0%s" (string .context.Params.season) (strings.TrimPrefix "B" .context.Params.seasonepisode) ) }}
+{{ else if eq (string .context.Params.seasonepisode) (string "0") }}
     {{/* Episode 0 are trailers */}}
-    {{ $.Scratch.Set "episodeNumber" (printf "Season %s Trailer" (string .Params.season) ) }}
-    {{ $.Scratch.Set "imageFolder" (printf "S0%sE0%s" (string .Params.season) (string .Params.seasonepisode) ) }}
+    {{ .context.Scratch.Set "episodeNumber" (printf "Trailer" ) }}
+    {{ .context.Scratch.Set "imageFolder" (printf "S0%sE0%s" (string .context.Params.season) (string .context.Params.seasonepisode) ) }}
 {{ else }}
     {{/* Regular episode */}}
-    {{ $.Scratch.Set "episodeNumber" (printf "Season %s Ep. %s" (string .Params.season) (string .Params.seasonepisode)) }}
-    {{ $.Scratch.Set "imageFolder" (printf "S0%sE0%s" (string .Params.season) (string .Params.seasonepisode) ) }}
+    {{ if eq .single "true" }}
```

Comment:
Passing the variable "single" into the partial means context must now be named and can't simply use the `.`